docs/version_bump_template.md

```diff
@@ -37,50 +71,49 @@ Available arguments to create your command with examples:
 * All flags are optional except for -cmd and -expectedValue but if you want to use one of them, you need to use all of them.
 
 
-Example of an execution with multiple values on k3s:
+Example of an execution with multiple values on k3s using versionbump tag:
 ```bash
 go test -timeout=45m -v -tags=versionbump  ./entrypoint/versionbump/... \
 -cmd "/var/lib/rancher/k3s/data/current/bin/cni, kubectl get pod test-pod -o yaml : | grep -A2 annotations, k3s -v" \
 -expectedValue "v1.2.0-k3s1,1M, v1.26" \
 -expectedValueUpgrade "v1.2.0-k3s1,1M, v1.27" \
--installVersionOrCommit INSTALL_K3S_VERSION=v1.27.2+k3s1 \
+-installVersionOrCommit 1.27.2+k3s1 \
```

Comment:
should be `v1.27...`?

Reply:
i had fixed somewhere else, doing now 
tks